src/main/java/com/revature/bean/BamUser.java

```diff
@@ -1,4 +1,4 @@
-package com.revature.batch.bean;
```

Comment:
packages should all be plural, so should be bean**s**
Oh, so that goes for repository->repositories, service->services, exception->exceptions, controller->controllers, logging ->(I have no idea!)